paper/user_study.tex

```diff
@@ -49,8 +49,128 @@ \subsection{Results}
+\subsection{Evaluation} %Keyuri
 % todo: show correctness of the tasks.
+\begin{table*}
+    \vspace{-10pt}
+    \centering
+    \small % Reduce font size
+    \setlength{\extrarowheight}{5pt} % Add extra vertical space
+    \renewcommand{\arraystretch}{1.5} % Adjust the vertical spacing between rows
+    \begin{tabular}{|c|p{0.4\linewidth}|p{0.4\linewidth}|}
+        \hline
+        \multicolumn{1}{|c|}{\multirow{2}{*}{}} & \multicolumn{1}{|c|}{Feedback} & \multicolumn{1}{|c|}{Resolution} \\
+        \cline{2-3}
+        \hline
+        User Study 1 & 1. The property value should not be autocorrected if the user enters an incorrect value. Instead, an error message or another way should be used to inform the user that their input is incorrect. & 1. To enhance user feedback, consider implementing a feature that displays a red underline beneath the property value when an error is detected. \\
+        \cline{2-3}
+        & 2. The ability to delete a property should be made easily accessible and user-friendly within the GUI panel. & 2. The deletion functionality has been implemented by adding a cross symbol next to each property, allowing users to easily delete a property by clicking on the cross symbol. \\
+        \cline{2-3}
+        & 3. Including a search functionality is essential to help users locate properties, especially within nested levels where finding specific property can be challenging. & 3.A search button has been implemented in the top toolbar to enable users to locate properties, even within nested levels. When a property is found, it is highlighted in yellow to make it easily identifiable. \\
+        \cline{2-3}
+        & 4. The Schema editor GUI panel contains extensive metadata, but the code editor panel remains empty. & 4. bla bla bla \\
+        \cline{2-3}
+        & 5.To maintain consistency and readability, it should be remove the use of different font styles of property (bold, purple colour, * with purple colour and so on)  in the file editor within the GUI panel. & 5. bla bla bla \\
+        \cline{2-3}
+        & 6. The cursor styling in the file editor within the GUI panel should be improved to ensure it doesn't resemble a link cursor. & 6. Cursor styling is chnaged. \\
+        \cline{2-3}
+        & 7. The cross symbol has been removed from the dropdown button for enum properties in the GUI panel as it is deemed unnecessary. & 7. The cross symbol has been removed from the dropdown button for enum properties in the GUI panel. \\
+        \cline{2-3}
+        & 8. If the type of a property is as "any" in the Schema Editor, it should not default to being interpreted as the "string" type. & 8. bla bla \\
+        \cline{2-3}
+        & 9. Validation error should not showed as warning symbol, proper indication needed for validation error rather than warning symbol. & 9.The warning symbol has been removed, and a red cross indication has been implemented to clearly indicate validation errors in the user interface. \\
```

Comment:
Shouldn't we put that right below the other point that addresses schema validation?
```suggestion
        & 9. Validation error should not showed as warning symbol, proper indication needed for validation error rather than warning symbol. & 9.The warning symbol has been removed, and a red cross indication has been implemented to clearly indicate validation errors in the user interface, in addition to the changed mentioned before \\
```